### ISSUE TITLE
Gaussian Splatting Color/Opacity export Issue

### ISSUE THREAD

**Issue by dlazares** [nerfstudio-project/nerfstudio#2689]
**Describe the bug**
The exported .ply looks washed out like it has an improper white balance or opacity. I couldn't get the viewer to work on the colab so I'm not sure if this is a training issue or an export issue. probably an export issue though with how the colors are saved out. I'm thinking it has something to do with pre-multiplied alpha or opacity issues.

Thanks for writing a nice and concise implementation of gaussian splatting!

**To Reproduce**
1. Go to the colab
2. Set dataset to dozer
3. set ns-train nerfacto to `gaussian-splatting` and run training
4. add a code block with something like
```%cd /content
!ns-export gaussian-splat --load-config /content/outputs/unnamed/gaussian-splatting/2023-12-18_033851/config.yml --output-dir /content/outputs/```
5. View exported .ply in a viewer like https://antimatter15.com/splat/

**Expected behavior**
Expect the exported ply to be the same colors as training images.

**Screenshots**

<img width="1355" alt="Screenshot 2023-12-17 at 8 36 16 PM" src="https://github.com/nerfstudio-project/nerfstudio/assets/10122282/a713f479-7fa7-43d8-848f-79960a40b081">

**Comment by hardikdava** [comment]
@tancik you can close this issue.

**Comment by pknmax** [comment]
is it possible to view the exported ply file in SIBR viewer?

**Comment by jb-ye** [comment]
@pknmax no, SIBR viewer's reader is more restrictive on the format data. The Inria version uses plyfile package to write ply file, while nerfstudio use open3d. Please note that plyfile is GPL 3+ licensed package.
